src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,72 @@
 import React from 'react';
+import { getUser } from '../../api/api';
+import { User } from '../../types/type';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+interface Props {
+  id: number;
+  clear: (id: number) => void;
+}
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+interface State {
+  user: User;
+  isUserLoaded: boolean,
+}
+
+export class CurrentUser extends React.Component<Props, State> {
+  state: State = {
+    isUserLoaded: true,
+    user: {
+      id: 0,
+      name: '',
+      email: null,
+      phone: null,
+    },
+  };
+
+  componentDidMount() {
+    this.loadUser();
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.id !== this.props.id) {
+      this.loadUser();
+    }
+  }
+
+  async loadUser() {
+    try {
+      const user = await getUser(this.props.id);
+
+      this.setState({ user, isUserLoaded: true });
+    } catch (error) {
+      this.setState({ isUserLoaded: false });
+      throw new Error(`User isn't loaded ${error}`);
```

Comment:
Don't throw error here, this will stop your site from working.
Display message instead of throwing another error.